.github/workflows/release-dry-run.yml

```diff
@@ -48,10 +48,11 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: ^1.16
+          go-version: '1.18'
```

Comment:
I think we want  go1.18, but not 100% sure

Reply:
100%